core/keyserver/keyserver.go

```diff
@@ -72,11 +72,15 @@ type WriteWatermark struct {
 
 // MutationLogs provides sets of time ordered message logs.
 type MutationLogs interface {
-	// Send submits an item to a random log.
+	// Send submits groups of item atomically to a random log.
```

Comment:
Please clarify. Does this send the whole group, or multiple sub-groups? Also, could use `mutations` instead of `items` to match the parameter name.

Reply:
``` // Send submits the whole group of mutations atomically to a random log.```